src/aat/resources/features/F-040/F-040.feature

```diff
@@ -0,0 +1,21 @@
+@F-040
+Feature: F-040: Get Case for Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-576
+  Scenario: must return successfully all and only the respective fields with READ access for Solicitors
+    Given a user with [an active Solicitor profile in CCD with a specific variation of ACLs on a case type]
```

Comment:
does `a specific variation of ACLs` make sense? how about `Given a user with [an active solicitor profile in CCD with limited permissions on certain fields for a given case type]`

Reply:
Updated the code to include "Given a user with [an active solicitor profile in CCD with limited permissions on certain fields for a given case type]".

Reply:
can you also update the one below this to something similar?